lib/template.html

```diff
@@ -111,27 +93,56 @@
   {% endif %}
 {% endif %}
 
+<script type="application/ld+json">
+  {
+    "@context" : "http://schema.org",
+
+{% if page.seo and page.seo.type %}
+    "@type" : {{ page.seo.type | jsonify }},
+{% elsif page.url == "/" %}
+    "@type" : "WebSite",
+{% elsif page.id %}
```

Comment:
There *must* be a better way to check if the current `page` is a **Post** :confounded: